Sharprompt/Forms/ListForm.cs

```diff
@@ -106,9 +106,14 @@ protected override bool TryGetResult(out IEnumerable<T> result)
                     case ConsoleKey.LeftArrow:
                     case ConsoleKey.RightArrow:
                     case ConsoleKey.Backspace:
-                    case ConsoleKey.Delete:
                         ConsoleDriver.Beep();
                         break;
+                    case ConsoleKey.Delete:
```

Comment:
If you only use the Delete key, it will conflict with the character deletion, so a combination key such as Ctrl+Delete would be better.

Reply:
That's a good point. I changed it so that the combination key `Ctrl+Delete` now triggers the delete.